utils.go

```diff
@@ -1,16 +1,17 @@
 package relay_grpc
 
 import (
-	"github.com/attestantio/go-builder-client/api/capella"
+	"github.com/attestantio/go-builder-client/api/deneb"
 	v1 "github.com/attestantio/go-builder-client/api/v1"
 	"github.com/attestantio/go-eth2-client/spec/bellatrix"
-	consensus "github.com/attestantio/go-eth2-client/spec/capella"
+	capella "github.com/attestantio/go-eth2-client/spec/capella"
+	consensus "github.com/attestantio/go-eth2-client/spec/deneb"
 	"github.com/attestantio/go-eth2-client/spec/phase0"
 	"github.com/holiman/uint256"
+	"math/big"
 )
 
-func CapellaRequestToProtoRequest(block *capella.SubmitBlockRequest) *SubmitBlockRequest {
-
+func DenebRequestToProtoRequest(block *deneb.SubmitBlockRequest) *SubmitBlockRequest {
```

Comment:
this function should ideally support capella blocks **_and_** deneb blocks, because we'll want to have a smooth transition for the fork
Something like 
```
VersionedRequestToProtoRequest(block *builderSpec.VersionedSubmitBlockRequest) *SubmitBlockRequest {
```

Reply:
Fixed